.github/config.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you 
+  include information such as the version of Pyne you're working with, the platform you
+  are operating on, the expected behavior, and the actual behavior you are bringing our 
+  attention to. The more deatil you provide, the better others in this community will 
+  be able to help you.
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Hi!  Welcome, and thanks for opening this pull request.  We have some guidelines for
```

Comment:
Should we link to guidelines? Do we have specific guidelines other than it must pass CI?

Reply:
We do not have a specific set of guidelines, I forgot to take that out. I can add a template for PRs, and change the line above to reflect that if we'd like more structure.